src/BylawsText.js

```diff
@@ -37,6 +42,22 @@ export default function BylawsText({ answers }) {
         .then(() => setIsTextCopied(true))
   }
 
+  const downloadBylaws = (text, fileType, fileName) => {
```

Comment:
Is there a reason why don't use a function declaration like  `function downloadByLaws(test, fileType, fileName){...}` ?
I think it makes it a little bit easier to see what is a variable and what's a function. Also, if you use function declarations you don't need to worry about the order of the code and you can put all the functions at the bottom, if you like.

Reply:
Good point :-) I changed it into a function declaration